lib/AnyEvent/HTTPD/Router.pm

```diff
@@ -14,6 +14,9 @@ sub new {
     my $class = ref($this) || $this;
     my %args  = @_;
 
+	# set allowed methods to GET until we get some routes
+    # why GET? because :verbs will need at least one real HTTP method
+    $args{allowed_methods} = ['GET'] unless $args{allowed_methods};
```

Comment:
I would take (or keep the defaults from super-class) that AE::HTTPD has in it's module. (which includes GET, and some more)
so you could leave out this line as well